### DISCUSSION TITLE
latest (22dec) update under linux (.all deb version) not recognized as update

### DISCUSSION THREAD

**Discussion by seth-space** [LMS-Community/slimserver#516] [General]
I tried to update my current Logitech Media Server Version: 8.0.0 - 1605619275 @ Tue Nov 17 14:59:27 CET 2020

Operating system: Debian - EN - utf8
Platform Architecture: x86_64-linux
Perl Version: 5.30.3 - x86_64-linux-gnu-thread-multi

To the latest available 8/0 http://downloads.slimdevices.com/nightly/8.0/lms/8b4b6da43b9349b3dc5b87eb08b350a55f540e08/logitechmediaserver_8.0.1~1608617100_all.deb
tried 8.1
http://downloads.slimdevices.com/nightly/8.2/lms/86471dfa3faaba64ea665d16af665e52677266bc/logitechmediaserver_8.2.0~1610887865_all.deb
tried 8.2
http://downloads.slimdevices.com/nightly/8.2/lms/86471dfa3faaba64ea665d16af665e52677266bc/logitechmediaserver_8.2.0~1610887865_all.deb

But its not seen as an update.
Only option is "remove" with Ubuntu software install.
Please advice..

**Comment by mherger** [comment]
Please try the latest release: http://downloads.slimdevices.com/LogitechMediaServer_v8.1.1/

And instead of the _all.deb I'd recommend you use the much smaller amd64 flavour instead, because you don't need all the files for ARM and other architectures in the all file.

**Reply by seth-space** [reply]
nope, same error.. btw its getting confusing all the different versions.

**Comment by mherger** [comment]
Did you try `sudo dpkg -i logitechmediaserver....deb`? I don't understand what you're trying to do, and how that fails... There are tens of thousands of .deb based installations out there.

**Reply by seth-space** [reply]
just replaced the standard Ubuntu installer with Gdebi and it seems to solve the problem So it looked like a local problem.
Solved!
